build/db_schemas/yugabyte/rid/downfrom-v1.0.1-drop-ybgin-indexes.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX s_cell_idx ON subscriptions USING ybgin (cells);
```

Comment:
If yugabyte is not deployed anywhere, might as well update v1.0.0 IMO. And it probably makes sense keeping doing so until the v1.0.0 is actually a true working v1 that is released.